deb2rpm.pl

```diff
@@ -102,6 +102,9 @@ sub write_spec {
     if (-e "debian/$package.install") {
       print $out "install_to_buildroot < %{rootdir}/debian/$package.install\n";
     }
+    if (-e "debian/$package.links") {
+      print $out "install_links_in_buildroot < %{rootdir}/debian/$package.links\n";
```

Comment:
I've followed the paradigm for installing the `logrotate` configuration file, which is also in the packaging environment.  I don't have a compelling reason either way beyond that though.